composer.json

```diff
@@ -20,7 +20,7 @@
         }
     ],
     "require": {
-        "php": ">=5.3",
+        "php": ">=7.4",
```

Comment:
Personally, I don't like to force the minor constraints as the package will often work but maybe throw deprecations. Would ^7.4 || ^8.0 be a possible way? Therefore we would allow everything above and including 7.4 and <= 9.0

```suggestion
        "php": "^7.4 || ^8.0",
```